### PR TITLE
made some changes to home screen

### PR DESCRIPTION
**Pull request by GregJoubert**
added expandable list views

### CODE REVIEW

File: PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java [hunk 1]
```diff
@@ -76,67 +118,190 @@ public void getPublicTimeline(RequestParams params, String token) throws JSONExc
...
+//            });
+//        } else
+//        {
+//            // The ViewPropertyAnimator APIs are not available, so simply show
+//            // and hide the relevant UI components.
+//            mProgressView.setVisibility(show ? View.VISIBLE : View.GONE);
+//            mHomeFormView.setVisibility(show ? View.GONE : View.VISIBLE);
+//        }
+//    }
+
```
Comment (JenCabral) [inline PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java]: Huge commented out section. Is it needed in the future?

**Comment by JenCabral** [comment]
See my comment about the commented out section. Otherwise looks good. Want to run it to see the changes visually.

File: PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java (hunk 1 above)
Comment (bmansell) [inline PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java]: I agree with Jen, is this necessary?

File: PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java [hunk 2]
```diff
@@ -10,15 +10,29 @@
 import android.text.TextUtils;
 import android.util.Log;
 import android.view.View;
+import android.widget.Button;
+import android.widget.ExpandableListView;
 import android.widget.TextView;
 import android.widget.Toast;
+
+import com.example.gregjoubert.partezapp.DataWrapper.Result;
+import com.example.gregjoubert.partezapp.DataWrapper.SearchResponse;
```
Comment (bmansell) [inline PartEzApp/app/src/main/java/com/example/gregjoubert/partezapp/HomeActivity.java]: We should probably do away with the gregjoubert packages before we develop further.

**Comment by bmansell** [comment]
Looks good! Thanks for changing the package naming convention.

**Pull request merged**